src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1796,6 +1796,47 @@ public static void TestJsonReaderUtf8SpecialString(string jsonString, string exp
             }
         }
 
+        [Fact]
+        public static void TestBOMWithSingleJsonValue()
+        {
+            Assert.ThrowsAny<JsonException>(() =>
+            {
+                ReadOnlySpan<byte> Utf8BomAndValue = new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' };
+                var json = new Utf8JsonReader(Utf8BomAndValue, true, default);
+                json.Read();
+            }
+            );
+
+            Assert.ThrowsAny<JsonException>(() =>
+            {
+                ReadOnlySpan<byte> Utf8BomAndValue = new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' };
```

Comment:
nit: camel case for locals (so lower case the u).

```suggestion
                ReadOnlySpan<byte> utf8BomAndValue = new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' };
```